v03_pipeline/lib/methods/relatedness.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+
+import hail as hl
+from gnomad.sample_qc.pipeline import filter_rows_for_qc
+
+from v03_pipeline.lib.model import ReferenceGenome
+
+
+# Stub port from https://github.com/broadinstitute/gnomad_qc/blob/9763d8a581665ed0be8bf2ea09d74e3cabc4806e/gnomad_qc/v2/resources/sample_qc.py#L40
+def qc_mt_path(reference_genome: ReferenceGenome) -> str:
+    ref_str = '.grch38' if reference_genome == ReferenceGenome.GRCh38 else ''
+    return f'gs://gnomad/sample_qc/mt/gnomad.joint.high_callrate_common_biallelic_snps.pruned{ref_str}.mt'
```

Comment:
are we sure the pipeline will have access to these resources?

Reply:
I believe it works as long as we set the "requester.pays" configuration for hail, which we do [here](https://github.com/broadinstitute/seqr-pipeline-airflow/blob/main/dags/dependencies/operators.py#L159-L161)

Reply:
it may be cheaper for us to store a copy than pay every time we load data though? Depending on what region those are stored in etc

Reply:
I wrote the code to import them and place cached copies in `seqr-reference-data-private` with the little "cached reference dataset query" pattern, so the import is at least reproducible and treated similarly to what we're doing with other supplemental tables.